aten/src/ATen/native/TensorShape.cpp

```diff
@@ -2133,8 +2133,22 @@ std::vector<Tensor> unbind(const Tensor& self, Dimname dim) {
 }
 
 std::vector<Tensor> meshgrid(TensorList tensors) {
+  return native::meshgrid(tensors, /*indexing=*/c10::nullopt);
+}
+
+std::vector<Tensor> meshgrid(TensorList tensors,
+                             c10::optional<c10::string_view> indexing) {
   int64_t size = tensors.size();
   TORCH_CHECK(size > 0, "meshgrid expects a non-empty TensorList");
+
+  if (!indexing.has_value()) {
+    TORCH_WARN_ONCE("In an upcoming release, it will be required to pass the "
+                    "indexing argument to torch.meshgrid.");
+    indexing = "ij";
+  } else {
+    TORCH_CHECK(*indexing == "ij", "Unsupported indexing: ", *indexing);
```

Comment:
For these error messages start with the name of the operation and let people know what the correct indexing mode/s is/are. For example:

"torch.meshgrid: only 'ij' indexing is supported at this time"

Reply:
should it still include the offending value?

Reply:
That'd be nice

Reply:
done.